src/main/java/edu/kis/powp/jobs2d/command/LoadFromFile.java

```diff
@@ -0,0 +1,15 @@
+package edu.kis.powp.jobs2d.command;
+
+public class LoadFromFile {
+    private CommandReader commandReader;
+    public void loadBaseOnExtension(String command, String fileExtension) {
+
+        switch(fileExtension) {
+            case "json":
+                commandReader = new LoadFromJson();
```

Comment:
This could be wrapped in a 'factory'. It would allow for more flexible readers management.